packages/okta-adapter/src/config.ts

```diff
@@ -1224,6 +1225,16 @@ const DEFAULT_TYPE_CUSTOMIZATIONS: OktaSwaggerApiConfig['types'] = {
       fieldsToOmit: DEFAULT_FIELDS_TO_OMIT.concat({ fieldName: '_links' }),
       fieldsToHide: [{ fieldName: 'id' }],
     },
+    deployRequests: {
+      // only modificaiton is supported
```

Comment:
1. typo nit
```suggestion
      // only modification is supported
```
2. we can still support additions if we first find the correct mapping id, no? (in a deploy that creates a new app won't we also want to support customizing the mapping?)

Reply:
yes thats a good point, I'll add it